cmd/saml2aws/main.go

```diff
@@ -70,6 +70,7 @@ func main() {
 	app.Flag("config", "Path/filename of saml2aws config file (env: SAML2AWS_CONFIGFILE)").Envar("SAML2AWS_CONFIGFILE").StringVar(&commonFlags.ConfigFile)
 	app.Flag("idp-account", "The name of the configured IDP account. (env: SAML2AWS_IDP_ACCOUNT)").Envar("SAML2AWS_IDP_ACCOUNT").Short('a').Default("default").StringVar(&commonFlags.IdpAccount)
 	app.Flag("idp-provider", "The configured IDP provider. (env: SAML2AWS_IDP_PROVIDER)").Envar("SAML2AWS_IDP_PROVIDER").EnumVar(&commonFlags.IdpProvider, "Akamai", "AzureAD", "ADFS", "ADFS2", "Browser", "GoogleApps", "Ping", "JumpCloud", "Okta", "OneLogin", "PSU", "KeyCloak", "F5APM", "Shibboleth", "ShibbolethECP", "NetIQ", "Auth0")
+	app.Flag("browser-type", "The configured browser type when the IDP provider is set to Browser. (env: SAML2AWS_BROWSER_TYPE)").Envar("SAML2AWS_BROWSER_TYPE").EnumVar(&commonFlags.BrowserType, "chrome", "chrome-beta", "chrome-dev", "chrome-canary", "msedge", "msedge-beta", "msedge-dev", "msedge-canary")
```

Comment:
Is there a reason why there's no support for Firefox? 👀

Reply:
```go
browser, err := pw.Chromium.Launch(launchOptions)
	if err != nil {
		return "", err
	}
```
It looks like it [always launches a Chromium browser](https://github.com/Versent/saml2aws/blob/f45c5f888cfdf9dcd1e8863e2301ff957e03d787/pkg/provider/browser/browser.go#L46-L49), so I'd guess that's the reason why.

Reply:
@TwiN, @zemberdotnet Added Firefox.

The original values were set based on the comment of the supported values by the playwright Channel; see: https://github.com/playwright-community/playwright-go/blob/8e8f670b5fa7ba5365ae4bfc123fea4aac359763/generated-structs.go#L441-L444

However, it totally made sense to me to add this capability, so I added support for Firefox and Webkit. (`msedge` is also supported using Chromium which is the default).

Reply:
@zemberdotnet thank you for noticing that.

The reason for this behaviour is that the `commonFlags` can be passed both in the `configure` or other steps such as `login`. In my case, i have set the `--browser-type` in the `configure` and then it was available in the login.

To support providing the flag in the login command, I removed the additional check for `commonFlags.IdpProvider == "Browser"` since the BrowserType is only used by the `Browser` Provider, the functionality is still the same.

Please check it again as it should work now.